packages/usa-button/src/usa-button.twig

```diff
@@ -2,20 +2,80 @@
   <div class="bg-base-darkest padding-1" style="max-width: fit-content">
 {% endif %}
 
+{% set icon %}
+  <!-- @TODO: Allow size to be passed as a argument. -->
+  <!-- usa-icon--size-{3-9} -->
+  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
+    <use xlink:href="./img/sprite.svg#{{ icon_name | default("add_circle_outline") }}"></use>
+  </svg>
+{% endset %}
+
 {% if is_demo and 'usa-button--big' not in modifier %}
-  <button type="{{ type }}" class="usa-button {{ modifier }}">Default</button>
-  <button type="{{ type }}" class="usa-button {{ modifier }} usa-button--hover">Hover</button>
-  <button type="{{ type }}" class="usa-button {{ modifier }} usa-button--active">Active</button>
-  <button type="{{ type }}" class="usa-button {{ modifier }} usa-focus">Focus</button>
-  <button type="{{ type }}" class="usa-button {{ modifier }}" disabled>Disabled</button>
-  <button type="{{ type }}" class="usa-button {{ modifier }}" aria-disabled="true">aria-disabled</button>
-  <button type="{{ type }}" class="usa-button {{ modifier }} usa-button--unstyled">Unstyled button</button>
+  <button type="{{ type }}" class="usa-button {{ modifier }}">
+    Default
+    {% if icon_included %}
+    {{ icon }}
+    {% endif %}
```

Comment:
Should we indent these for improved legibility? 
```suggestion
    {% if icon_included %}
       {{ icon }}
    {% endif %}
```

>**Warning**
> Github suggestion won't let me hit the tab button so this is just a few spaces. Don't commit this example.

Reply:
## Indents

Resolving for now since the indent note doesn't apply anymore - Reworked in 87fc766.

## Spacing

@amyleadem this might be okay for now. Open to discussing further in a separate enhancement. We'd have to solution how to target styles when icon is placed before/after text.